src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -2296,7 +2299,6 @@ def aks_check_acr(cmd, client, resource_group_name, name, acr):
                         "args": ["-v6", acr],
                         "stdin": True,
                         "stdinOnce": True,
-                        "tty": True,
```

Comment:
Remove this since usually there is no chance for interactivity.

At the same time, it will also improve the stability of the test case. In the past, it often happened that the last line of output was lost.